src/components/ReportActionItemIOUAction.js

```diff
@@ -0,0 +1,89 @@
+import React, {Component} from 'react';
+import {View} from 'react-native';
+import PropTypes from 'prop-types';
+import {withOnyx} from 'react-native-onyx';
+import ONYXKEYS from '../ONYXKEYS';
+import ReportActionItemIOUQuote from './ReportActionItemIOUQuote';
+import ReportActionPropTypes from '../pages/home/report/ReportActionPropTypes';
+import ReportActionItemIOUPreview from './ReportActionItemIOUPreview';
+import Navigation from '../libs/Navigation/Navigation';
+import ROUTES from '../ROUTES';
+
+const propTypes = {
+    // All the data of the action
+    action: PropTypes.shape(ReportActionPropTypes).isRequired,
+
+    // The associated chatReport
+    chatReportID: PropTypes.number.isRequired,
+
+    // Should render the preview Component?
+    shouldDisplayPreviewComp: PropTypes.bool.isRequired,
+
+    /* --- Onyx Props --- */
+    // ChatReport associated with iouReport
+    chatReport: PropTypes.shape({
+        // The participants of this report
+        participants: PropTypes.arrayOf(PropTypes.string),
+    }),
+
+    // Session info for the currently logged in user.
+    session: PropTypes.shape({
+        // Currently logged in user email
+        email: PropTypes.string,
+    }).isRequired,
+};
+
+const defaultProps = {
+    chatReport: {},
+};
+
+class ReportActionItemIOUAction extends Component {
+    constructor(props) {
+        super(props);
+
+        this.launchIOUDetailsModal = this.launchIOUDetailsModal.bind(this);
+    }
+
+    /**
+     *
+     * Launch the IOU Details Modal, using data from the report action
+     */
+    launchIOUDetailsModal() {
+        Navigation.navigate(ROUTES.getIouDetailsRoute(
+            this.props.chatReportID, this.props.action.originalMessage.IOUReportID,
+        ));
+    }
+
+    render() {
+        const isDMChat = this.props.chatReport.participants.length === 1;
```

Comment:
NAB, I think it would be better to make this `hasMultipleParticipants` since we are not really using the `DM` language anywhere else.

Reply:
I switched to this as the logic seemed a bit simpler, but that is a good point. Will revert.